osgbuild/main.py

```diff
@@ -314,7 +314,7 @@ def parse_cmdline_args(argv):
     parser = OptionParser(header)
     parser.add_option(
         "-a", "--autoclean", action="store_true",
-        help="Clean out the following directories before each build: "
+        help="(default) Clean out the following directories before each build: "
```

Comment:
The conventions for `--help` text and the conventions for manpages are not the same. While prose is more appropriate for a manpage, I think there is something to be gained here for using a concise string that can be identified at a glance. Looking at a few examples, such as `tar --help`, however, I don't see any place where "(default)" is at the beginning, so I'll fix that.